### PR TITLE
Join us CRM integration and error states

### PR DESCRIPTION
**Pull request by alanmoo**
See #15 and #85

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ TODO.txt
 node_modules/
 dest/
 source/json/temp/
+/.env
```
Comment (gvn) [inline .gitignore]: This can come out

File: source/js/basket-signup.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+var url = `https://www.mozilla.org/en-US/newsletter/`;
```
Comment (gvn) [inline source/js/basket-signup.js]: Add a TODO comment about making this dynamic based on env

File: env/default.json [hunk 3]
```diff
@@ -1,3 +1,4 @@
 {
-  "virtualRoot": "/"
+  "virtualRoot": "/",
+  "basketUrl": "https://www.mozilla.org/en-US/newsletter/"
```
Comment (gvn) [inline env/default.json]: Remove for now since this isn't hooked up

File: env/staging.json [hunk 4]
```diff
@@ -1,3 +1,4 @@
 {
-  "virtualRoot": "/network/"
+  "virtualRoot": "/network/",
+  "basketUrl": "https://www.allizom.org/en-US/newsletter/"
```
Comment (gvn) [inline env/staging.json]: Remove for now since this isn't hooked up

**Comment by alanmoo** [comment]
Oh, I never even tried using that button, I thought it wasn't hooked up yet.

**Pull request merged**